airbyte-config/config-models/src/main/java/io/airbyte/config/Configs.java

```diff
@@ -540,6 +540,11 @@ public interface Configs {
    */
   String getJobKubeNamespace();
 
+  /**
```

Comment:
See my other comment - I think we can avoid adding this new method to the interface altogether, and instead prefer Micronaut injection